Src/FluentAssertions/Types/TypeSelector.cs

```diff
@@ -183,6 +183,32 @@ public TypeSelector ThatAreNotClasses()
         return this;
     }
 
+    /// <summary>
+    /// Determines wheter the type is a struct (a value type but not an Enum)
+    /// </summary>
+    /// <returns></returns>
+    public TypeSelector ThatAreStruct()
```

Comment:
What about renaming to `ThatAreStructs`?

Reply:
make sense